code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -459,17 +462,61 @@
 	return ..()
 
 /mob/living/simple_animal/bot/screwdriver_act(mob/living/user, obj/item/I)
+	. = TRUE //Must be true or we attempt to stab the bot | Sets standard return value for this code block
+
 	if(user.a_intent == INTENT_HARM)
 		return ..()
 	if(locked)
-		to_chat(user, "<span class='warning'>The maintenance panel is locked.</span>")
-		return TRUE // Must be true or we attempt to stab the bot
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
+		return
 
-	open = !open
 	I.play_tool_sound(src)
+	open = !open
 	to_chat(user, "<span class='notice'>The maintenance panel is now [open ? "opened" : "closed"].</span>")
+	return
+
+/mob/living/simple_animal/bot/wrench_act(mob/living/user, obj/item/I)
+	. = TRUE //Must be true or we attempt to stab the bot | Sets standard return value for this code block
+
+	if(user.a_intent != INTENT_HELP)
+		return ..()
+	if(locked) //ID-Lock
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
+		return
+	if(!open) //Screwdriver panel
+		to_chat(user, "<span class='warning'>The maintenance panel is screwed in place.</span>")
+		return
+	if((bot_type != SEC_BOT) && (bot_type != GRIEF_BOT)) //Should only affect security bots and griefsky
```

Comment:
```suggestion
	if(bot_type != SEC_BOT && bot_type != GRIEF_BOT) //Should only affect security bots and griefsky
```

Reply:
I thought dm would behave weirdly without the parenthesis but I guess I thought too little of it